src/main/scala/io/sdkman/changelogs/ToolkitMigrations.scala

```diff
@@ -0,0 +1,50 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "073")
+class ToolkitMigrations {
+
+  val CandidateName = "toolkit"
+
+  @ChangeSet(
+    order = "001",
+    id = "001-add_toolkit_migrations_candidate",
+    author = "iot-technology"
+  )
+  def migration001(implicit db: MongoDatabase): Candidate = {
+    Candidate(
+      candidate = CandidateName,
+      name = "Toolkit Migrations",
+      description =
+        "Toolkit is a client command line tool that supports multiple IoT protocols. for example, MQTT and COAP protocols" +
+          "Provide a user optimized client tool for IoT developers and learners",
+      websiteUrl = "https://github.com/IoT-Technology/IoT-Toolkit"
+    ).insert()
+  }
+
+  @ChangeSet(
+    order = "003",
+    id = "003-add_toolkit_0.4.0_as_default",
+    author = "iot-technology"
+  )
+  def migration003(implicit db: MongoDatabase) =
```

Comment:
```suggestion
  @ChangeSet(
    order = "002",
    id = "002-add_toolkit_0.4.0_as_default",
    author = "iot-technology"
  )
  def migration002(implicit db: MongoDatabase) =
```

Reply:
:heartpulse:   thank for your suggestion

sdkman will get better